### ISSUE TITLE
Popup opens on Streaming Site VOE

### ISSUE THREAD

**Issue by HeroFruit** [AdguardTeam/PopupBlocker#304]
Hello, on the streaming page VOE with the video address https://stevenimaginelittle.com/e/f4gybdxnzzpx the popup still opens an ad page.

**Comment by alexx7311** [comment]
@HeroFruit Hello! We couldn't reproduce the problem. Please clear the browser's cache & cookies or open the link in incognito mode. Does that help?

**Comment by HeroFruit** [comment]
hi. I tried to clear the cache and cookies and even different Browsers and tried it on my phone. Sill happening. I made an clip https://youtu.be/zhVkxE4JKyE .
I just klick on the videoplayer and its opens a adpage. This pages gets loaded.
![Screenshot 2023-12-07 155705](https://github.com/AdguardTeam/PopupBlocker/assets/119861301/9ec89576-421a-4703-bbbb-9bcb8f1aef15)
i usually use Chrome only with Adguard extensions: Adguard Extra and this Popup blocker. And only Adguard Filters with DNS and basic Privacy protection settings. 
i hope i can help you
